frontend/src/components/user/UserCreateModal.vue

```diff
@@ -29,12 +29,13 @@ const authzStore = useAuthZStore();
 // State
 const loading: Ref<boolean> = ref(false);
 const searchTag: Ref<string> = ref('');
-const selectableGroups: Ref<Array<GroupName>> = ref([]);
+const selectableGroups: Ref<Map<string, GroupName>> = ref(new Map());
 const selectedGroup: Ref<GroupName | undefined> = ref(undefined);
-const selectedParam: Ref<string | undefined> = ref(undefined);
+const selectedParam: Ref<string | undefined> = ref('Last name');
 const selectedUser: Ref<User | undefined> = ref(undefined);
 const users: Ref<Array<User>> = ref([]);
 const visible = defineModel<boolean>('visible');
+const yarsGroups: Ref<Array<Group>> = ref([]);
```

Comment:
Doesn't look like this is used outside of `onMounted`. I think we can removed the state value and just make it a `const` in `onMounted`.